lib/canvas/checks/valid_liquid_check.rb

```diff
@@ -41,6 +41,7 @@ def register_tags!
       register_tag("currency_switcher", ::Liquid::Tag)
       register_tag("json", ::Liquid::Block)
       register_tag("variant_pricing", ::Liquid::Tag)
+      register_tag("experience_date_search", ::Liquid::Tag)
```

Comment:
I think this needs to use `::Liquid::Block`, which matches the class the ExperienceDateSearchTag inherits from [here](https://github.com/easolhq/easol/blob/799542309059e813076e69384c4f789394672529/app/tags/experience_date_search_tag.rb#L8)

Reply:
Good catch @ianmooney it was a naive assumption. Thanks! 🙏